DatadogInternal/Sources/NetworkInstrumentation/TraceID.swift

```diff
@@ -6,8 +6,6 @@
 
 import Foundation
 
-public typealias SpanID = TraceID
```

Comment:
The fact that SpanID is 64-bit and TraceID is 128-bit feels not obvious to me just by name.

For example in the b3 definition it's either 64 or 128 for TraceID and always 64-bit for SpanID.
For W3C and OTel they seem to be aligned.

Maybe worth just naming them more agnostically, like `Identifier64` and `Identifier128`

Reply:
Actually scratch that. OTel lib is doing exactly that:
https://github.com/open-telemetry/opentelemetry-swift/blob/812e3fca27a0bbc6a281b0b54546694251b54243/Sources/OpenTelemetryApi/Trace/TraceId.swift
https://github.com/open-telemetry/opentelemetry-swift/blob/812e3fca27a0bbc6a281b0b54546694251b54243/Sources/OpenTelemetryApi/Trace/SpanId.swift